src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,58 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  filterOption: string,
+  query: string,
+  setFilterOption: (filterOption: string) => void,
```

Comment:
Use type of setState function
```suggestion
  setFilterOption: Dispatch<SetStateAction<FilterOption>>,
```